src/components/PopoverListItem/__snapshots__/PopoverListItem.test.ts.snap

```diff
@@ -0,0 +1,73 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<PopoverListItem /> Active story renders snapshot 1`] = `
+<div
+  class="popover-list-item popover-list-item--active"
+>
+  <div
+    class="popover-list-item__icon"
+  >
+    <svg
+      aria-hidden="true"
+      class="icon"
+      fill="currentColor"
+      height="1.5rem"
+      style="--icon-size: 1.5rem;"
+      width="1.5rem"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <use
+        xlink:href="test-file-stub#check"
+      />
+    </svg>
+  </div>
+  Active Row with Icon
+</div>
+`;
+
+exports[`<PopoverListItem /> Default story renders snapshot 1`] = `
+<div
+  class="popover-list-item"
+>
+  <div
+    class="popover-list-item__no-icon"
+  />
+  Default list item
+</div>
+`;
+
+exports[`<PopoverListItem /> Disabled story renders snapshot 1`] = `
+<div
+  class="popover-list-item popover-list-item--disabled"
+>
+  <div
+    class="popover-list-item__no-icon"
+  />
+  Disabled
+</div>
```

Comment:
This is the specific element that's triggering the a11y failure, and I think it's happening because this is a `div` no `aria-disabled` attribute. (It may also need a `role`; I don't know exactly how the test works.)

Disabled interactive elements are exempt from color contrast requirements in WCAG, so it makes sense that this story is failing the test since it's not marked up as disabled.

Source: https://webaim.org/articles/contrast/
>WCAG 2.0 defines four types of "incidental" text that are not required to meet the contrast requirements.
>
>- Inactive: An inactive element, like a disabled Submit button ( Submit ), is identified visually by its lower-contrast state.
>- Pure decoration: Decorative text that is not meant to be read. An example of this might be a picture of a bookshelf on a library homepage. The titles of the books are not meant to be read by the user.
>- Not visible to anyone: Text that is meant to be hidden, like an invisible [skip link](https://webaim.org/techniques/skipnav/) would not need to meet any contrast requirements until it becomes visible.
>- Part of a picture that contains significant other visual content: Text that is not an important part of the information in the image, like a name tag on the shirt of a person in a photo of a party, does not need to meet any contrast requirements.